Tools/wasm/Setup.local.example

```diff
@@ -5,6 +5,7 @@ audioop
 _bz2
 _crypt
 _decimal
+nis
```

Comment:
This change looks unrelated to the rest of the PR.

Reply:
Tehnically, yes, but while testing this it led to https://github.com/python/cpython/issues/102851 which this would close.